CHARTER.md

```diff
@@ -10,33 +10,37 @@ The Arcalot Round Table *(ART)* consists of:
-
-An Arcalot Round Table member *(including chairs)* that does not participate in round table matters (voting) *without reason and notice* for 3 times in a row leaves the round table automatically as we can assume they are not interested in being a member of the ART anymore. This ensures that we do not have a corpus of inactive members with voting rights.
+### ART Participation
 
 Meetings do not count as necessary round table matters as we want to include people in all stages of life and across all timezones. We therefore refrain from any synchronous or undocumented decision making.
 
 Additionally, as stated in the [Arcalot Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md), chairs can remove an ART member if their behavior is clearly disruptive to the community.
 
+
+### Chair Participation
+
+Chairs are expected to remain actively involved in the community. An Arcalot Round Table chair that does not participate in round table matters (voting) *without reason and notice* for 3 times in a row leaves the chairs automatically as we can assume they are not interested in the responsibilities of the chair position anymore.
+
+
 ## Decision Proposals
 
-Decisions can be started *only asychronously* in the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) via submitting a pull request with a new file to [art-decisions/proposals](art-decisions/proposals). You can use the included `template.md` file as a guideline. Decision proposals can only be submitted by ART members. If you would like to encourage discussion around a decision before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not add any changes to your proposal.
+Decisions can be started *only asychronously* in the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) via submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). You can use the included [template file](art-decisions/proposals/template.md) as a guideline. Decision proposals can only be submitted by ART members. If you would like to encourage discussion around a decision before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not add any changes to your proposal.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
+Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a minimum of 4 votes is required with 75% approval majority among the voters.
```

Comment:
"started" seems like the wrong word.  "approved" might work; or, perhaps, replace "a vote is started where" with "approval requires" (and drop "is required" and "approval").  Replace "among" with "of".

Reply:
", acceptance (of the proposal) requires a minimum of four votes with 75% approval among those voting"??